src/classes/UTIL_ACCT_Naming.cls

```diff
@@ -35,296 +35,652 @@
-            for (String lastName : mapLastNameContacts.keySet()) {
-                lnCounter++;
-                //Adding connector only if a last name has more than two contacts
-                //Starting at the second contact
-                if (nameContactFieldsString != null
-                    && nameContactFieldsString != ''
-                    && mapLastNameContacts.keySet().size() > 1 
-                    && lnCounter > 1) {
-                    if (lnCounter != mapLastNameContacts.keySet().size()) {
-                        nameContactFieldsString += ', ';
-                    } else {
-                        nameContactFieldsString += Label.defaultNamingConnector + ' ';
-                    }
-                }
 
-                for (String nameString : listNameStrings) {
-                    String conNameString = '';
-                    if (UTIL_Describe.isValidField('Contact', nameString.toLowerCase())) {
-                        if (nameString == 'lastname') {
-                            conNameString += lastname;
-                        } else {
-                            Integer conCounter = 0;
-                            for (Contact con : mapLastNameContacts.get(lastName)) {
-                                conCounter++;
-                                if (con.get(nameString) != null) {
-                                    if (conNameString != null
-                                            && conNameString != ''
-                                            && mapLastNameContacts.get(lastName).size() > 1
-                                            && conCounter > 1) {
-                                        if (conCounter != mapLastNameContacts.get(lastName).size()) {
-                                            conNameString += ', ';
-                                        } else {
-                                            conNameString += ' ' + Label.defaultNamingConnector + ' ';
-                                        }
-                                    }
-                                    conNameString += con.get(nameString);
-                                }
-                            }
-                        }
-                        if (conNameString != null
-                                && conNameString != '') {
-                            nameContactFieldsString += conNameString;
-                        }
-                    } else {
-                        nameContactFieldsString += nameString;
-                    }
+        return ns.namePrefix + stringAccountName(finalAccountName, ns) + ns.nameSuffix;
+    }
+
+    /*******************************************************************************************************
+    * @description This method is reponsbile for concatenating all additional Salutations, FirstNames, and LastName
+    * into a string to return the final Account name.
+    * @param cons is the list of Contacts in the current trigger context.
```

Comment:
Thanks for updating the parameter list! This is just `con` now though, and is no longer a list of Contacts.